TestThermo/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.9.0" targetFramework="net45" />
-  <package id="NUnit3TestAdapter" version="3.9.0" targetFramework="net45" />
-  <package id="System.ValueTuple" version="4.4.0" targetFramework="net45" />
+  <package id="NUnit" version="3.10.1" targetFramework="net461" />
+  <package id="NUnit3TestAdapter" version="3.10.0" targetFramework="net461" />
+  <package id="System.ValueTuple" version="4.4.0" targetFramework="net45" requireReinstallation="true" />
```

Comment:
does it matter if this is targeting net45?

Reply:
Good question. I'm not sure there's another option for this one. It does build...